lib/tara/installer.rb

```diff
@@ -63,6 +64,45 @@ def bundle_gems
       end
     end
 
+    def find_installed_gems
+      definition = Bundler::Definition.build('lib/vendor/Gemfile', 'lib/vendor/Gemfile.lock', false)
+      return [] unless definition.has_optional_groups?
+      @without_groups.each do |group|
+        definition.add_optional_group(group)
+      end
+      definition.specs.each_with_object([]) do |gem_spec, specs|
+        if gem_spec.full_gem_path.start_with?(Bundler.bundle_path.to_s) # Local gem
+          specs << {
+            :name => gem_spec.name,
+            :full_name => gem_spec.full_name,
+            :full_gem_path => gem_spec.full_gem_path,
+            :spec_file => gem_spec.spec_file,
+            :bin_dir => gem_spec.bin_dir,
+            :relative_path => Pathname.new(gem_spec.full_gem_path).relative_path_from(Bundler.bundle_path).to_s,
+          }
```

Comment:
Seems like only `full_gem_path`, `relative_path` and `spec_file` are used, so I'm assuming the others are unnecessary?